spec/feature/styleguide_spec.rb

```diff
@@ -0,0 +1,9 @@
+RSpec.feature 'Styleguide', :js do
+  scenario 'all components are properly documented' do
```

Comment:
Exceptions are thrown when the component doesn't have a description and at least one example.